RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptions/ManageSubscriptionsViewModel.swift

```diff
@@ -181,17 +182,19 @@ private extension ManageSubscriptionsViewModel {
             do {
                 guard let purchaseInformation = self.purchaseInformation else { return }
                 let productId = purchaseInformation.productIdentifier
-                self.customerCenterActionHandler?(.refundRequestStarted(productId))
+                self.actionBridge.handleAction(.public(.refundRequestStarted(productId)))
+
                 let status = try await self.purchasesProvider.beginRefundRequest(forProduct: productId)
                 self.refundRequestStatus = status
-                self.customerCenterActionHandler?(.refundRequestCompleted(status))
+                self.actionBridge.handleAction(.public(.refundRequestCompleted(status)))
```

Comment:
I am not convinced about the fact that legacy actions need to be wrapped under public now 🤔 @ajpallares

Reply:
Do you refer to the name `public` or to the fact that legacy actions are wrapped?

If it's about the naming, I agree that `public` is not the best name for this. As @facumenzella suggested, `legacy` is a better name.

Reply:
The thing is that they are not really legacy, they are still as valid as the new ones but these need to trigger an extra callback

Reply:
take a look at [`0a85523` (#4844)](https://github.com/RevenueCat/purchases-ios/pull/4844/commits/0a85523271246fc7bcd690e7d216bf8634419c00)